service/build.gradle

```diff
@@ -74,7 +74,7 @@ dependencies {
 	testImplementation 'io.findify:s3mock_2.13:0.2.6'
 
 	pitest 'com.arcmutate:base:1.3.2'
-	pitest 'com.arcmutate:pitest-git-plugin:2.2.1'
+	pitest 'com.arcmutate:pitest-git-plugin:2.1.0'
```

Comment:
the version of pitest was downgraded as there seems to be some version incompatibility with the newest Gradle